### PR TITLE
build: add postgresql db config (with sqlalchemy)

### PR DESCRIPTION
**Pull request by raphodn**
### What

- install the `psycopg2-binary` & `sqlalchemy` dependencies. Remove `peewee`
- update the .env & app config to pass the postgres db config parameters
- connect to the db on app startup
- update documentation

TODO
- docker config ?

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -15,3 +15,10 @@ SENTRY_DNS=
 
 # Log level to use, DEBUG by default in dev
```
Comment (raphael0202) [inline .env.example:16]: Locally we should have a ".env" file ready for local development, so you can rename this file ".env"

File: .env.example [hunk 2]
```diff
@@ -15,3 +15,10 @@ SENTRY_DNS=
...
 LOG_LEVEL=DEBUG
+
+# Postgres database
+POSTGRES_DB_NAME=open_prices
```
Comment (raphael0202) [inline .env.example:20]: We can keep it simple locally (postgres/postgres/postgres for DB name, user and password), as we use Docker usually and it's for local dev only

File: INSTALL.md [hunk 3]
```diff
@@ -22,6 +22,12 @@ source venv/bin/activate
 # install
 pip install -r requirements.txt
 
+# create a PostgreSQL database
+psql -c "CREATE USER open_prices_team WITH PASSWORD 'password'"
```
Comment (raphael0202) [inline INSTALL.md:26]: You can skip this if you keep the defaults

**Review by raphael0202 — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (raphodn) [reply .env.example:16]: ok ! 
- i deleted `.env.example`
- remove `.env` from the gitignore
- simplified the install doc

**Pull request merged**